CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Fix #3818: adding missing throws to launderThrowable
 * Fix #3859: refined how a deserialization class is chosen to not confuse types with the same kind
 * Fix #3936: Kubernetes Mock Server .metadata.generation field is an integer
+* Fix #3950: make sure modifications on jdk websocket done by interceptors are not lost
```

Comment:
Hi @shawkins,
This is an unreleased feature, so maybe we should remove this entry and add the appropriate entry to the `#### New Features` section WDYT?

Reply:
I had the same thought about removing the fix entry.  There are already some entries in the migration guide and the faq about the client usage.

Either with #3887 or another issue we'll need to add the jdk to the release build.

Reply:
Ok I'll remove it